opensrp-reveal/src/main/java/org/smartregister/reveal/viewholder/GroupedListableViewHolder.java

```diff
@@ -0,0 +1,15 @@
+package org.smartregister.reveal.viewholder;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+
+import org.smartregister.reveal.contract.GroupListContract;
+import org.smartregister.view.ListContract;
+import org.smartregister.view.viewholder.ListableViewHolder;
+
+public abstract class GroupedListableViewHolder<T extends ListContract.Identifiable> extends ListableViewHolder<T> implements GroupListContract.GroupedAdapterViewHolder<T> {
```

Comment:
This class has no added methods, not sure why its added

Reply:
It's possible to have it directly implemented but the goal is to move this piece of code to be shareable with other implementations that might need to use grouped lists